test/integration/visitor_can_edit_cart_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class VisitorCanEditCartTest < ActionDispatch::IntegrationTest
+  test "visitor can remove an item from his or her cart" do
+    category_1 = Oil.create(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
```

Comment:
Useless assignment to variable - `item_1`. Did you mean `item_2`?<br>Line is too long. [106/80]